### ISSUE TITLE
Pre- vs post-breeding census issues

### ISSUE THREAD

**Issue by patrickbarks** [jonesor/Rage#54]
I still have trouble with this distinction (apparently I'm in [good company](https://doi.org/10.1016/j.ecolmodel.2019.03.011)).

I think the Rage functions that involve vital rates of fecundity (`vr_` fns, `perturb_vr`, and possibly `mpm_to_mx`; using devel branch names here) effectively assume a post-breeding census design. I'm not sure whether it's possible to add pre-breeding census functionality for these, since in a pre-breeding model the newborn survival rate only appears in the fecundity transition — so the birth rate and newborn survival rate are not independently estimable.

I can't decide whether this problem also applies to `mpm_to_mx` (and therefore `makeLifeTable`), or whether mx trajectories from pre- and post-breeding MPMs are comparable.

Not sure how to resolve this. For now I guess we can just add some documentation to the relevant functions.

**Comment by jonesor** [comment]
Yes, let's just add some documentation to the relevant functions - @RobSalGo could you take a look?

**Comment by jonesor** [comment]
Closeed this issue with last commit
